intro-HPC/ch_troubleshooting.tex

```diff
@@ -108,6 +108,63 @@ \section{Issues connecting to login node}
 and include the output of that command in the message.
 \fi
 
+\section{Security warning about invalid host key}
+\label{sec:security-warning-invald-host-key}
+
+\ifmacORlinux
+
+If you get a warning that looks like the one below, it may be possible
+that someone is trying to intercept the connection between you and the \hpc.
+
+\begin{prompt}
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
+Someone could be eavesdropping on you right now (man-in-the-middle attack)!
+It is also possible that a host key has just been changed.
+The fingerprint for the ECDSA key sent by the remote host is
+SHA256:1MNKFTfl1T9sm6tTWAo4sn7zyEfiWFLKbk/mlT+7S5s.
+Please contact your system administrator.
+Add correct host key in ~/.ssh/known_hosts to get rid of this message.
+Offending ECDSA key in ~/.ssh/known_hosts:%\strong{21}%
+ECDSA host key for %\loginnode{}% has changed and you have requested strict checking.
+Host key verification failed.
+\end{prompt}
+
+You will need to remove the line it's complaining about (in the example, line 21).
+To do that, open \verb|~/.ssh/config| in an editor, and remove the line. This
+results in \verb|ssh| ``forgetting'' the system you are connecting to.
+
+After you've done that, you'll need to connect to the \hpc again.
+See \autoref{sec:warning-message-new-host} to verify the fingerprints.
+\strong{It's important to verify the fingerprints. If they don't match,
+do \emph{not} connect and contact \hpcinfo instead.}
+
+\else
+
+If you get a warning box that looks like the one below, it may be possible
+that someone is trying to intercept the connection between you and the system you are connecting to.
+Another possibility is that the host key of the system you are connecting to has changed.
```

Comment:
This should be out of the if (but drop the `box` and change `may be possible` to `is possible`). And remove the almost some block of text under `\ifmacORlinux` of course